packages/statistics-module/__tests__/statistics-module.test.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const statisticsModule = require('..');
+
+describe('statistics-module', () => {
+    it('needs tests');
+});
```

Comment:
Are we gonna put all tests inside the `__tests__` folder? or what's gonna be the configuration for tests?

Reply:
The folder was created by the template, tests should be stay together with the logic. We will remove this folder :open_file_folder:

Reply:
We will remove this folder for the moment, and we are going to add it following the links-module structure in a following ticket.